DeviceCode/Targets/OS/CMSIS_RTOS/CMSIS_RTOS.settings

```diff
@@ -33,5 +33,15 @@
                Condition="!EXISTS('$(SPOCLIENT)\CMSIS\ARM.CMSIS.pdsc')"
                Text="Missing CMSIS installation at: $(SPOCLIENT)\CMSIS\*"
                />
+        <!-- Check CMSIS version -->

```

Comment:
:innocent: 

I made an explicit check ( == 4.3.0) due to RTX directory change. For the future releases, the check could be relaxed to ">= x.y", if there are no breaking changes (solutions that do not use RTX compile just fine, there are no breaking changes in CMSIS 4.3).